### PR TITLE
Create occupation standard from data_import spreadsheet

### PR DESCRIPTION
**Pull request by littleforest**
Asana ticket: https://app.asana.com/0/1203289004376659/1203289004376679/f

This creates a new occupation standard from the Details tab of a DataImport file.

The data being imported has this format:

Registration State | OA or SAA | Sponsor Name | Occupation Title | Existing Title | Onet Code | RAPIDS Code | Type | Term (in months) | Minimum OJT Hours | Maximum OJT Hours | Minimum RSI Hours | Maximum RSI Hours | Ratio of Apprentice to Journeyworker | Probationary Period 
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- 
CA | OA |   | HUMAN RESOURCE SPECIALIST | Career Development Technician | 13-1071.01 | 1057CB | Competency-based | 12 |   |   |   |   | 5:1 | 3

### CODE REVIEW

**Comment by littleforest** [comment]
@FerPerales @jenlu33 this is ready for review.

File: app/models/occupation_standard.rb [hunk 1]
```diff
@@ -2,13 +2,20 @@ class OccupationStandard < ApplicationRecord
...
   belongs_to :registration_agency
   belongs_to :organization, optional: true
+  belongs_to :data_import, optional: true
 
   has_many :related_instructions
   has_many :wage_steps
 
   delegate :rapids_code, to: :occupation, allow_nil: true
 
+  enum occupation_type: [:time, :competency, :hybrid], _suffix: :based
```
Comment (FerPerales) [inline app/models/occupation_standard.rb:12]: I didn't know about `_suffix`. Very neat 💯

File: db/schema.rb [hunk 2]
```diff
@@ -210,12 +224,12 @@
     t.string "title"
     t.string "description"
     t.uuid "occupation_standard_id", null: false
+    t.integer "default_hours"
     t.integer "minimum_hours"
     t.integer "maximum_hours"
     t.integer "sort_order"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "default_hours"
```
Comment (FerPerales) [inline db/schema.rb:218]: Wasn't this already fixed? 🤔

**Review by FerPerales — APPROVED** [review]
Looks good. Left a couple of comments.

Question: is there a way we can reduce the number of migrations? Not a big deal, but it was hard to keep track of the changes

File: db/schema.rb (hunk 2 above)
Comment (littleforest) [reply db/schema.rb:218]: Then it got unfixed in #40

**Comment by littleforest** [comment]
> Question: is there a way we can reduce the number of migrations? Not a big deal, but it was hard to keep track of the changes

Yes, let me see if I can remove some of those `int4range` ones that I ended up changing to int min/max fields.

**Review by FerPerales — APPROVED** [review]

**Pull request merged**